apps/aecontract/src/aect_state_tree.erl

```diff
@@ -34,3 +36,7 @@ insert_contract(_Contract, Tree) ->
     %% PLACEHOLDER
     Tree.
 
+-spec root_hash(tree()) -> {ok, aeu_mtrees:root_hash()} | {error, empty}.
+root_hash(#contract_tree{}) ->
+    {error, empty}.
```

Comment:
Addition of TODO comment here welcome.

Reply:
Ah it is empty anyway so there is not way that can be forgotten. (Other functions have PLACEHOLDER for remembering.)